vnext/Microsoft.ReactNative/packages.config

```diff
@@ -0,0 +1,12 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
`packages.config` is deprecated in favor of `PackageReference` entries.

Please add/update dependencies in `Microsoft.ReactNative.vcxproj` instead.

Reply:
Nice catch. Done :)